crates/shared/src/order_quoting.rs

```diff
@@ -486,9 +486,22 @@ impl Now for DateTime<Utc> {
     }
 }
 
-/// Standard validity for a quote: Quotes are stored only as long as they are
-/// valid.
-const STANDARD_QUOTE_VALIDITY_SECONDS: i64 = 60;
+pub struct Validity {
+    pub eip1271_onchain_quote: Duration,
+    pub presign_onchain_quote: Duration,
+    pub standard_quote: Duration,
+}
+
+#[cfg(test)]
+impl Default for Validity {
+    fn default() -> Self {
+        Self {
+            eip1271_onchain_quote: Duration::seconds(60),
+            presign_onchain_quote: Duration::seconds(60),
```

Comment:
Should these two be `600`?

Reply:
They were 60s in the tests, but you are right probably better to make 600 (the actual default config) by default